src/util/dbAggregations.ts

```diff
@@ -0,0 +1,18 @@
+import connectMongo from "./connectMongo";
+import Movie from "../../models/movie";
+
+export async function getMovie(idQuery: string) {
+
+    await connectMongo();
+  
+    const movie = Movie.aggregate([
+      { $match: { title: idQuery } },
```

Comment:
To follow up on my previous comment about ID vs title.

This is risky, because there may be more than one movie that has the same title. Instead I would use the ID, perhaps together with the slugified title. With that pattern, the code could look something like:

```ts
export async function getMovie(slug: string) {

    await connectMongo();

    const [id, ...rest] = slug.split('-');

    const movie = Movie.aggregate([
      { $match: { id: id } },
```